lldb/source/Host/posix/MainLoopPosix.cpp

```diff
@@ -384,6 +313,21 @@ void MainLoopPosix::ProcessReadObject(IOObject::WaitableHandle handle) {
     it->second(*this); // Do the work
 }
 
+void MainLoopPosix::ProcessSignals() {
+  std::vector<int> signals;
```

Comment:
(Uneducated question) Is int actually the correct type for signals, and not an unsigned value?